dts/msm8929-samsung-r01.dts

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+        qcom,msm-id =   <268 0>,
+                        <269 0>,
+                        <270 0>,
+                        <271 0>;
```

Comment:
The indentation on this line is weird (spaces instead of tabs). Also, in general there should be exactly one msm-id that is used for a particular SoC. Can you try to figure out which one is the correct one?

The easiest way might be to check the lk2nd log using

```
$ fastboot oem lk_log && fastboot get_staged <someoutputfile>
```

Then look for the `Board:` line, e.g. on SM-A500FU:

```
[130] Board: platform: 206, foundry: 0x0, platform_version: 0x10000, platform_hw: 0x8, platform_subtype: 0x0, target: 0x10008, baseband: 0x0, platform_hlos_subtype: 0x0
[150] pmic_info[0]: type: 0x1000b, version: 0x20000, target: 0x2000b
```

MSM8916 has qcom,msm-id 206.

Reply:
For the J7008:
`[130] Board: platform: 268, foundry: 0x0, platform_version: 0x10000, platform_hw: 0x1, platform_subtype: 0x0, target: 0x10001, baseband: 0x0, platform_hlos_subtype: 0x0
`